### PR TITLE
Product Information Skeleton

### PR DESCRIPTION
**Pull request by Jmok19927**
Set up the skeleton for my section, and added basic functionality on swapping around styles. 

addressed merge conflicts, needed to capitalize the Q in one of the QnA imports

### CODE REVIEW

File: src/components/ProductDetails/ProductDetails.jsx [hunk 1]
```diff
@@ -1,17 +1,46 @@
...
+      styles: [],
+      currentStyle: {},
     }
   }
 
+  componentDidMount() {
+    axios.defaults.headers.common['Authorization'] = config.TOKEN;
+    axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/products/${this.props.id}/styles`).then((data) => {
+      // console.log('style data in Style selector',data.data.results[0])
+      this.setState({styles: data.data.results, currentStyle:data.data.results[0]})
```
Comment (ntjohns10) [inline src/components/ProductDetails/ProductDetails.jsx:22]: Should the current style just be the first style in the styles array, or should it be the style with 'default?: true' key value pair?

**Review by ntjohns10 — COMMENTED** [review]
Looks good, I left a comment about which style the 'current style' defaults to.  DM me with questions.

File: src/components/ProductDetails/ProductDetails.jsx (hunk 1 above)
Comment (Jmok19927) [reply src/components/ProductDetails/ProductDetails.jsx:22]: Good catch, I've adjusted it to prioritize the 'default?' value, and then fall back to the first entry in the array if there is no default? value in the commits below

**Comment by ntjohns10** [comment]
Looks great to me!

**Pull request merged**